adhocracy4/images/fields.py

```diff
@@ -1,19 +1,11 @@
 from django.db import models
 from django.conf import settings
 from django.utils.translation import ugettext_lazy as _
-from django.utils.functional import lazy
+from django.utils.text import format_lazy
 
 from . import forms, validators
 
 
-# FIXME: backportet from django 1.11. remove this after update
```

Comment:
Opin is and will stay on Django 1.8 for the next months. If Django 1.8 support is removed it should also be made clear in the `setup.py`.

Reply:
Reverted the commit.